addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -687,9 +698,14 @@ import wUtils from '@website/js/utils';
                     return value.name === '';
             }
 
-            const format = value.includes(':')
-                ? localization.dateTimeFormat
-                : localization.dateFormat;
+            let format = "";
+            let currentDate = new Date();
```

Comment:
I prefer variable declaration in a scope of use. May be it should be declared & initialize in switch case.

Reply:
Variable needs to be initialized and declared according to format of "value" variable, so i used the existing if-else condition which uses localization based on same condition. after that line value variable is converted into integer and deciding condition does not exist anymore inside switch-case, this can be reused if we want to add another date based condition visibility option in future.